test-site/webpack.config.js

```diff
@@ -24,13 +23,6 @@ module.exports = [{
   resolve: {
     extensions: [ '.tsx', '.ts', '.js' ],
   },
-  plugins: [
-    new CopyPlugin({
-      patterns: [
-        { from: 'src/index.html' },
-      ],
```

Comment:
Without this, we wouldn't be able to view the stuff in src/index.html since the test-site `npm run serve` command serves the dist folder. Maybe we should keep it?

Reply:
oh I see, sometimes (like today) I just open the file directly in my browser, but that's fair!